### DISCUSSION TITLE
WORST STAFF!

### DISCUSSION THREAD

**Discussion by guardyourheart** [hlky/stable-diffusion-webui#1052] [General]
WORST STAFF! CAN'T EVEN HELP WITH SINGLE PROBLEM! I"M SWTICHING TO Voldemort BYE AND GOOD LUCK DEVS!

**Comment by codedealer** [comment]
Somebody warn Harry Potter!

**Comment by sanscolour** [comment]
I'm surprised he didn't ask for a refund.

**Comment by ghost** [comment]
Consequences will NEVER be the same!!!
